docs/userguide.md

```diff
@@ -276,19 +277,24 @@ you can patch role `kubefed-role` in the kubefed namespace instead.
 
 ### Disabling federation of an API type
 
-You can disable propagation of an API type using the
-`kubefedctl` command.
+You can disable propagation of an API type by editing its `FederatedTypeConfig`
+resource:
 
 ```bash
-kubefedctl disable <FederatedTypeConfig Name>
+kubectl patch --namespace <KUBEFED_SYSTEM_NAMESPACE> federatedtypeconfigs <NAME> \
+    --type=merge -p '{"spec": {"propagation": "Disabled"}}'
 ```
 
-This command sets the `propagationEnabled` field in the `FederatedTypeConfig`
-associated with this target API type to `false`, which will prompt the sync controller for the target API type to be stopped.
+This patch command sets the `propagation` field in the `FederatedTypeConfig`
+associated with this target API type to `Disabled`, which will prompt the sync
+controller for the target API type to be stopped.
 
-If you want to permanently disable federation of the target API type, pass the
-`--delete-crd` flag to remove the `FederatedTypeConfig` and federated type CRD created by
-`enable`.
+### Deleting federation of an API type
+
+If you want to permanently delete federation of the target API type by
```

Comment:
The wording of this is a bit awkward.  Suggest separating  from the implementation i.e.

'If you want to permanently disable federation of a target type, use `kubefed disable`. This will remove the FTC that configures federation of the type. If supplied with the optional `--delete-crd` flag, the command will also remove the federated type CRD and all of its instances.'

Maybe the above removes the need for a warning?

Reply:
Deleting all of the instances is not yet implemented and tracked by #829. I've reworded this section to be more clear and removed the warning since it's technically no longer accurate.